### ISSUE TITLE
Style/RedundantParentheses conflict with Lint/AmbiguousRegexpLiteral

### ISSUE THREAD

**Issue by timhughes** [rubocop-hq/rubocop#6309]
I believe I have found 2 conflicting cops on the code 

I get a `Lint/AmbiguousRegexpLiteral` for the following code 
```
 its(:stdout) { should match /Consul v0\.7/ }
```

If i do as instructed i get a `Style/RedundantParentheses`
```
its(:stdout) { should match (/Consul v0\.7/) }
```
--------

## Expected behavior
Maybe there is a different way to fix this issue but the docs don't explain it. One set of docs say add some parentheses and then the other docs say remove them 

## Actual behavior

Cannot get it to pass rubocop

## Steps to reproduce the problem

Create the following code and run it through rubocop and you get a `Lint/AmbiguousRegexpLiteral`
```
describe command('/usr/local/bin/consul version') do
  its(:stdout) { should match /Consul v0\.7/ }
end
```
fix it by adding parentheses as per instructions https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/AmbiguousRegexpLiteral

```
describe command('/usr/local/bin/consul version') do
  its(:stdout) { should match (/Consul v0\.7/) }
end
```
Run again and you get a `Style/RedundantParentheses: Don't use parentheses around a literal.` 
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RedundantParentheses which says to remove the parentheses.

then my brain goes into a very tight infinite loop until I need to go and grab a coffee.

## RuboCop version
```
rubocop -V
0.49.1 (using Parser 2.5.1.2, running on ruby 2.4.3 x86_64-linux)
```

**Comment by Drenmi** [comment]
When putting a space between the method call and the argument, the parens don't bind to the method call. 🙂 This is what `Lint/AmbiguousRegexpLiteral` wants:

```ruby
its(:stdout) { should match(/Consul v0\.7/) }
```

which passes all checks.
